src/api.ts

```diff
@@ -63,37 +330,157 @@ export default class VoucheryIO {
 
       throw new Error(errorMsg);
     }
+    return method === 'DELETE'
```

Comment:
We don't need status for http request because we will have error thown by the package if the request has the error (4xx or 5xx).
Axios will throw the error if the status is 4xx series or 500 series.